Spigot-Server-Patches/0247-Biomes-can-be-swapped-in-config.patch

```diff
@@ -0,0 +1,224 @@
+From 88fac529695c47a275fed215349fcd95889ba0ef Mon Sep 17 00:00:00 2001
+From: June <junehanabi@gmail.com>
+Date: Wed, 18 Oct 2017 20:05:43 -0500
+Subject: [PATCH] Biomes can be swapped in config
+
+
+diff --git a/src/main/java/com/destroystokyo/paper/PaperConfig.java b/src/main/java/com/destroystokyo/paper/PaperConfig.java
+index f5cb9799..a9637da0 100644
+--- a/src/main/java/com/destroystokyo/paper/PaperConfig.java
++++ b/src/main/java/com/destroystokyo/paper/PaperConfig.java
+@@ -23,6 +23,7 @@ import org.bukkit.configuration.InvalidConfigurationException;
+ import org.bukkit.configuration.file.YamlConfiguration;
+ import co.aikar.timings.Timings;
+ import co.aikar.timings.TimingsManager;
```

Comment:
Please revert all formatting changes to this file.

getMap change belongs in the Paper Config patch